docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md

```diff
@@ -0,0 +1,203 @@
+​
+import (
+	"fmt"
+	"log"
+​
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+)
+​
+func main() {
+​
+	queueURL := "https://sqs.ap-south-1.amazonaws.com/xxxxxxxxxxxx/input"
+	region := "ap-south-1"
+	config := &aws.Config{
+		Region:      &region,
+		Credentials: credentials.NewStaticCredentials("xxxxxxxxxxxx", "xxxxxxxxxx", ""),
+	}
+​
+	sess, err := session.NewSession(config)
+	if err != nil {
+		log.Panic("Error while creating session")
+	}
+	svc := sqs.New(sess)
+​
+	for i := 100; i < 200; i++ {
+		msg := fmt.Sprintf("Hello Msg %v", i+1)
+		_, err := svc.SendMessage(&sqs.SendMessageInput{
+			DelaySeconds: aws.Int64(10),
+			MessageBody:  &msg,
+			QueueUrl:     &queueURL,
+		})
+		if err != nil {
+			log.Panic("Error while writing message")
+		}
+	}
+}
+```
+
+Since the go program uses SQS queue, we need to create the input queue to run the above program.
+
+We are now ready to package this code and create a function so that we can execute it later.
+Following commands will create a environment, package and function.
+Verify that build for package succeeded before proceeding.
+
+```sh
+$ fission env create --name goenv --image fission/go-env --builder fission/go-builder
+$ zip -qr sqs.zip *
+$ fission package create --env goenv --src sqs.zip
+Package 'sqs-zip-xpoi' created
+$ fission fn create --name producerfunc --env goenv --pkg sqs-zip-xpoi --entrypoint Handler
+$ fission package info --name sqs-zip-xpoi
+Name:        sqs-zip-xpoi
+Environment: go-sqs
+Status:      succeeded
+Build Logs:
+Building in directory /usr/src/sqs-zip-xpoi-1bicov
+```
+
+### Consumer function
+
+The consumer function is nodejs function which takes the body of the request, appends a "Hello" and returns the resulting string.
+
+```js
+module.exports = async function (context) {
+    console.log(context.request.body);
+    let obj = context.request.body;
+    return {
+        status: 200,
+        body: "Hello "+ JSON.stringify(obj)
+    };
+}
+```
+
+Let's create the environment and function:
+
+```bash
+$ fission env create --name nodeenv --image fission/node-env
+$ fission fn create --name consumerfunc --env nodeenv --code hellosqs.js
+```
+
+### Connecting via trigger
+
+We have both the functions ready but the connection between them is the missing glue.
+Let's create a message queue trigger which will invoke the consumerfunc every time there is a message in `input` queue.
+The response will be sent to `output` queue and in case of consumerfunc invocation fails, the error is written to `error` queue.
+
+```bash
+$ fission mqt create  --name sqstest --function consumerfunc --mqtype aws-sqs-queue --topic input --resptopic output --mqtkind keda --errortopic error --maxretries 3 --metadata queueURL=https://sqs.ap-south-1.amazonaws.com/xxxxxxxx/input --metadata awsRegion=ap-south-1 --cooldownperiod=30 --pollinginterval=5 --secret awsSecrets
```

Comment:
Can we add a comment here pointing to https://keda.sh/docs/2.0/scalers/aws-sqs/ for all metadata fields used?

Reply:
added description for the field required and removed unwanted